packages/amplication-server/src/core/auth/auth.resolver.ts

```diff
@@ -54,6 +54,14 @@ export class AuthResolver {
         });
   }
 
+  @Mutation(() => String)
+  @UseGuards(GqlAuthGuard)
+  async completeSignupWithBusinessEmail(
```

Comment:
is this related to the PR?

Reply:
of course..it's the resolver

Reply:
I don't know why it seems like I did it. It's what Shimi did. Maybe something with the merges...

Reply:
yes...it's the resolver of my new endpoing

Reply:
Then the naming is misleading as it might assume that is any business email and not preview only